extensions-core/hdfs-storage/src/main/java/io/druid/storage/hdfs/HdfsDataSegmentPusher.java

```diff
@@ -112,16 +112,17 @@ public DataSegment push(File inDir, DataSegment segment) throws IOException
       );
       Path outDir = new Path(String.format("%s/%s", config.getStorageDirectory(), storageDir));
       if (!fs.rename(tmpFile.getParent(), outDir)) {
-        if(!fs.delete(tmpFile.getParent(), true)){
+        if (!fs.delete(tmpFile.getParent(), true)) {
           log.error("Failed to delete temp directory");
```

Comment:
it will be nice to include the tmpFile name in the log message as well, so that manual cleanup can be done easily.